cmd/playground/static/assets/app.js

```diff
@@ -2091,8 +2069,8 @@ $(function () {
                         }
 
                         // store the port for newly created function
-                        var httpPort = _.get(pollResult, 'spec.httpPort', 0);
-                        _.set(selectedFunction, 'spec.httpPort', httpPort);
+                        var httpPort = _.get(pollResult, 'status.httpPort', 0);
+                        _.set(selectedFunction, 'status.httpPort', httpPort);
```

Comment:
Maybe we should place `status.httpPort` in a constant somewhere?

Reply:
It's not really a constant, it's just the lodash way of accessing variables by name. Even at the cost of repetition, this does not feel like a constant to me